spec/requests/portfolio_items_rbac_spec.rb

```diff
@@ -26,4 +27,37 @@
       expect(response).to have_http_status(403)
     end
   end
+
+  context "when user does not have RBAC write portfolios access" do
+    before do
+      allow(RBAC::Access).to receive(:new).with('portfolio_items', 'read').and_return(access_obj)
+      allow(access_obj).to receive(:process).and_return(access_obj)
+
+      allow(RBAC::Access).to receive(:new).with('portfolios', 'write').and_return(block_access_obj)
+      allow(block_access_obj).to receive(:process).and_return(block_access_obj)
+    end
+
+    it 'returns a 403' do
+      post "#{api("1.0")}/portfolio_items/#{portfolio_item1.id}/copy", :headers => default_headers
+
+      expect(response).to have_http_status(403)
+    end
+  end
+
+  context "when user has RBAC write portfolios access" do
+    let(:portfolio_access_obj) { instance_double(RBAC::Access, :accessible? => true, :owner_scoped? => true, :id_list => [portfolio.id.to_s]) }
```

Comment:
@lindgrenj6 I think the :owner_scoped could be false here since the Portfolio and Portfolio Items could be owned by anyone and shared with others unlike the orders and order items or progress messages which are owner specific.